test/process.jl

```diff
@@ -0,0 +1,12 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+import Base.Cmd
+
+#equality tests for Cmd
```

Comment:
couldn't seem to find any tests of the code in process so made this - let me know if there's a better place to put this